kratos/includes/gid_io.h

```diff
@@ -265,6 +265,12 @@ class KRATOS_API(KRATOS_CORE) GidIO
         mGidMeshContainers.push_back( TMeshContainer(
                                           GeometryData::KratosGeometryType::Kratos_Triangle2D6,
                                           GiD_Triangle, "Kratos_Triangle2D6_Mesh" ) );
+        mGidMeshContainers.push_back(TMeshContainer(
```

Comment:
I don't know if GiD supports these geometries. @jginternational ?

Reply:
Actually GiD can't read these geometries. However, I added this part for consistency with the other geometries.
Do I replace it with an error message?

Reply:
GiD allows triangles with 3 and 6 nodes. There are no plans to add more triangular geometries

Reply:
In that case, I would remove, or we will have not functional GiD outputs. An alternative is to "patch" the new geometries into old geometries. So for example, divide it into linear triangles.

Reply:
@loumalouomega @mnabideltares 
We have this list of valid elements in GiD and their connectivities: [https://gidsimulation.atlassian.net/wiki/spaces/GRM/pages/2385547733/Element+type](https://gidsimulation.atlassian.net/wiki/spaces/GRM/pages/2385547733/Element+type)

Reply:
For the moment I'd remove as @loumalouomega suggests. In the future we can (in an independent PR) think about patching the visualization of such high order geometries by using low order ones.

Reply:
Removed.
I also think it has to be done in an independent PR and @loumalouomega  and @rubenzorrilla  suggested.